Project.toml

```diff
@@ -1,12 +1,12 @@
 name = "MAT"
 uuid = "23992714-dd62-5051-b70f-ba57cb901cac"
-version = "0.6.0"
+version = "0.6.1"
 
 [deps]
 BufferedStreams = "e1450e63-4bb3-523b-b2a4-4ffa8c0fd77d"
 HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
-Libz = "2ec943e9-cfe8-584d-b93d-64dcb6d567b7"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+CodecZlib = "944b1d66-785c-5afd-91f1-9de20f533193"
```

Comment:
We need to migrate to adding bounds on all dependencies (https://discourse.julialang.org/t/announcement-automatic-merging-for-the-general-registry/29961), if you don't want to tackle them all can you at least add one for CodecZlib that you know works?